src/containers/BooksForm.js

```diff
@@ -1,19 +1,52 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../actions/index';
 
 function BooksForm() {
+  const bookTitle = useRef();
+  const bookCategory = useRef();
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
+
+  function handleChangeTitle() {
+    const title1 = bookTitle.current.value;
+    setTitle(title1);
+  }
+
+  function handleChangeCategory() {
+    const category1 = bookCategory.current.value;
+    setCategory(category1);
+  }
+
+  function handleSubmit() {
+    const book = {
+      bookId: uuidv4(),
+      title,
+      category,
+    };
+    dispatch(addBook(book));
+    setTitle('');
+    setCategory('');
+    bookTitle.current.value = null;
+  }
+
+  function handleForm(event) { event.preventDefault(); }
```

Comment:
### [Optional]
- It would be better if we use ES6 syntax instead of the function keyword. We are using `const` which is ES6 so it would be better if we follow with the consistency